### ISSUE TITLE
Y.Effects -min vs -raw

### ISSUE THREAD

**Issue by apipkin** [bialecki/yui3-gallery#4]
node.fade({ fade : { filter : function(e){ ... } });

works in -min but not in -raw

**Comment by bialecki** [comment]
Yes, this is a problem.  Somehow the minified file got out of sync with the raw file.  I'm going to be fixing this tonight.  Thanks for pointing out.

**Comment by bialecki** [comment]
Turns out this is an issue with the YUI build process. They're figuring it out and fixing it up as well speak.  I'll let you know once it's good to go.
